package/gluon-luci-mesh-vpn-fastd/i18n/fr.po

```diff
@@ -0,0 +1,37 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
+"Project-Id-Version: PACKAGE VERSION\n"
+"PO-Revision-Date: 2015-08-19 20:20+0100\n"
+"Last-Translator: Bernot Tobias <tqbs@airmail.cc>\n"
+"Language-Team: French\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid ""
+"In performance mode, no encryption is used. This usually allows for higher "
+"throughput, but the data exchanged over your node is not protected against "
+"eavesdropping."
+msgstr ""
+"En Mode  performance, aucune encryption est utilisé. Ceci permet un débit plus "
+"élevé, mais toutes les données échangées ne sont pas protégées contre l'écoute"
+
+msgid ""
+"In security mode, the mesh VPN uses an encrypted tunnel to connect to the "
+"VPN servers. The encryption ensures that it is impossible for your internet "
+"access provider to see what data is exchanged over your node."
+msgstr ""
+"En mode sécurtié, le mesch VPN utilise un tunnel encrypté pour te connecter "
+"aux serveurs VPN. L'encryption permet de rendre impssosible a votre FAI de "
```

Comment:
~~impssosible~~`impossible`
 ... FAI? Well I'm not living in France. Never heard this acronym for ISP:-)